### PR TITLE
Images

### PR DESCRIPTION
**Pull request by vincerubinetti**
image

This PR is not done. Still need to add the workflow images and the "at a glance" animation.

Temp live preview:
https://www.vincentrubinetti.com/manubot/

### CODE REVIEW

File: docs/getting-started.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="../styles.css" />
+  </head>
+  <body>
+    <header style="height: 100%">
+      <video width="100%" controls autoplay muted>
+        <source src="getting-started.mp4" type="video/mp4" >
```
Comment (dhimmel) [inline docs/getting-started.html]: This video is 94 MB... a bit large for tracking with git, close to GitHub's limit of 100 MB. Can we just use the mp4 from Figshare so we don't have to track it in this repo?

Git LFS would also be another alternative (never used it with GitHub Pages). We could use LFS for all binary files. Update: this does not work well as per https://github.com/git-lfs/git-lfs/issues/1342

**Comment by vincerubinetti** [comment]
@dhimmel This is ready to go.

File: index.html [hunk 2]
```diff
@@ -162,45 +172,117 @@
...
+          <div>
+            <b>Binding thermodynamics of host-guest systems with SMIRNOFF99Frosst 1.0.5 from the Open Force Field Initiative</b>
+            <i>D. Slochower, et al</i>
+          </div>
+        </a>
+        <a href="https://greenelab.github.io/deep-review/" class="example_image">
+          <img src="images/deep-review.png" />
+          <div>
+            <b>Opportunities and obstacles for deep learning in biology and medicine: 2019 update</b>
+            <i>C. Greene, et al</i>
```
[the remark this answers is not in the document]
Comment (agitter) [reply index.html]: @vincerubinetti we should change this before merging

File: index.html [hunk 3]
```diff
@@ -162,45 +165,117 @@
...
+          <div>
+            <b>Systematic integration of biomedical knowledge prioritizes drugs for repurposing</b>
+            <i>D. Himmelstein, et al</i>
+          </div>
+        </a>
+        <a href="https://trang1618.github.io/tpot-fss-ms/" class="example_image">
+          <img src="images/tpot.png" />
+          <div>
+            <b>Scaling tree-based automated machine learning to biomedical big data with a feature set selector</b>
+            <i>T. Ching, et al</i>
```
Comment (agitter) [inline index.html]: `T. Le` instead of `T. Ching`

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (vincerubinetti) [reply index.html]: Whoops, I had changed the wrong one. Should be fixed now.

File: docs/getting-started.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      <video width="100%" controls autoplay muted>
+        <source src="https://ndownloader.figshare.com/files/14874821" type="video/mp4" >
+      </video>
+      <h2>Coming Soon</h2>
+      <i class="fas fa-drafting-compass fa-lg"></i>
+      <br>
+      <div>
+        We are currently in the process of redesigning the Manubot workflow to be simpler and easier to use.
+        We are waiting until this is complete to continue writing the docs, as it will significantly change how Manubot works.
+        In the meantime, please refer the <a href="https://github.com/manubot/rootstock/blob/master/USAGE.md">documentation</a>, <a href="https://github.com/manubot/rootstock/issues">discussions</a>, and <a href="https://github.com/manubot/rootstock">template manuscript</a> on <a href="https://github.com/manubot">GitHub</a>.
```
Comment (agitter) [inline docs/getting-started.html]: This would be a good place to link to https://github.com/manubot/try-manubot/ to encourage experimentation.

**Comment by vincerubinetti** [comment]
@agitter It should be fixed now, give it a try.

**Comment by agitter** [comment]
@vincerubinetti yes, the getting started page works for me now

File: docs/getting-started.html (hunk 4 above)
Comment (dhimmel) [reply docs/getting-started.html]: Yeah. Great idea.

**Pull request merged**